src/main/java/uk/gov/ons/census/casesvc/model/dto/CollectionCase.java

```diff
@@ -17,7 +17,7 @@
   private Address address;
   private OffsetDateTime actionableFrom;
   private Boolean receiptReceived;
-  private Boolean refusalReceived;
+  private String refusalReceived;
```

Comment:
I know we do the same for other field but curious why we're not using the enum in the DTO?

Reply:
I would like to see the enum used